docusaurus/docs/e2e-test-a-plugin/locating-elements.md

```diff
@@ -0,0 +1,43 @@
+---
+id: locating-elements
+title: Locating elements for E2E tests
+description: How to locate elements on the page
+draft: true
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - e2e
+  - example test
+sidebar_position: 30
+---
+
+# Locating elements for E2E tests
```

Comment:
Just sharing a thought rather than requesting a change. The term "locating" seems to make sense from a playwright perspective, I just wonder if we need to call out more that it is the playwright nomenclature given the use of selectors -> selecting within what we've provided thus far.

Reply:
Yeah the fact that `locator` is a thing in Playwright is a bit confusing. 

I wonder if it would make sense to avoid the word `locate` in the first section, and save it for the second heading when Playwright is being mentioned the first time?  Then we can link the Playwright docs, making it obvious that it's an actual entity in the Playwright world.

Reply:
See [this](https://github.com/grafana/plugin-tools/pull/745/commits/c8d59ef7fddac4ce9b22438359031360bd85c58c) commit. Let me know what you think.

Reply:
The challenge is also in the page title itself. I like the changes in that commit though

Reply:
Ignoring Grafana or Playwright precedent, I would think of "Selecting UI elements" or "Navigating to UI elements" for the function, but I would defer to you on which (including leave as is) is the best.

Reply:
and my prior approval on the PR stands without changing anything

Reply:
Yeah I like `Selecting UI elements` - let's go with that.